python/src/mas/cli/install/argBuilder.py

```diff
@@ -59,6 +59,7 @@ def buildCommand(self) -> str:
         command += f" --mas-instance-id {self.getParam('mas_instance_id')}"
         command += f" --mas-workspace-id {self.getParam('mas_workspace_id')}"
         command += f" --mas-workspace-name \"{self.getParam('mas_workspace_name')}\"{newline}"
+        command += f" --allow-special-chars \"{self.getParam('mas_special_characters')}\"{newline}"
```

Comment:
This needs to be in a conditional, otherwise it will be added even when the user didn't set it ... it also doesn't have a vaue ... e.g.  this is what you are adding atm:

`--allow-special-chars true` or `--allow-special-chars true`

But we need logic that says add ``--allow-special-chars` only if `self.getParam('mas_special_characters') == true` ... you'll find many examples of this kind of logic for flags that don't have a value elsewhere in the script.